cmd/cluster/core/destroy.go

```diff
@@ -166,7 +188,13 @@ func DestroyCluster(ctx context.Context, hostedCluster *hyperv1.HostedCluster, o
 		}
 	}
 
-	o.Log.Info("Successfully destroyed cluster and infrastructure", "namespace", o.Namespace, "name", o.Name, "infraID", o.InfraID)
+	o.Log.Info("Successfully destroyed cluster", "namespace", o.Namespace, "name", o.Name, "infraID", o.InfraID)
+	if hciFound {
+		o.Log.Info(fmt.Sprintf("You can now delete the HostedClusterInfrastructure resource (Namespace/Name) %s/%s\nUse the command \"kubectl -n %s delete hci %s\"",
+			o.Namespace, o.InfraID, o.Namespace, o.InfraID))
+	} else {
+		o.Log.Info("Successfully destroyed infrastructure", "namespace", o.Namespace, "name", o.Name, "infraID", o.InfraID)
```

Comment:
Should this say "Successfully destroyed the hosted cluster and its infrastructure"?

Reply:
So the first log message says, successfully destroyed cluster. Ln 191

Then if HCI is present, it says, you can delete the HCI using this command.
Otherwise it destroys the infrastructure and logs it on Ln 196.

I could just add Successfully destroyed cluster to both Ln 193 and Ln 196, if you think it reads better.

Reply:
Sorry My mistake. I missed the first log message